include/odp/api/spec/traffic_mngr.h

```diff
@@ -605,6 +605,36 @@ odp_tm_t odp_tm_find(const char            *name,
  */
 int odp_tm_capability(odp_tm_t odp_tm, odp_tm_capabilities_t *capabilities);
 
+/** Start a TM system.
+ *
+ * odp_tm_start() needs to be used to start an already created / found TM system
+ * By default, all the TM systems are in stopped state.
+ *
+ * @param odp_tm  The odp_tm_t value of the TM system to be started.
+ * @return        0 upon success, < 0 upon failure.
+ */
+int odp_tm_start(odp_tm_t odp_tm);
```

Comment:
Documentation format could be update to match other APIs. TM API is not the best reference on how to document APIs.

- The first sentence misses a period (".") at the end: "... already created / found TM system."
- The first parameter could be simply "tm" and documented as TM system (handle):  "@param tm  TM system to be started"
- Simple return codes are usually documented with retval (return is used for more complex documentation):
```
@retval 0 on success
@retval <0 on failure